src/index.ts

```diff
@@ -807,7 +836,7 @@ class ThreadPool {
           taskInfo.workerInfo.taskInfos.delete(taskInfo.taskId)
           if (!taskInfo.workerInfo.taskInfos.size) {
             this._removeWorker(taskInfo.workerInfo)
-            this._ensureMaximumWorkers()
+            // this._ensureMaximumWorkers()
```

Comment:
are you sure about this? 👀

Reply:
Yes, we should not spawn workers, we should let the scheduler figures it out!

Reply:
Yes, we should not spawn workers, we should let the scheduler figures it out!